include/common/enum.inc

```diff
@@ -885,82 +885,84 @@ E(MalformedTable, 0x011C, "malformed table")
 
 // Validation phase
 // @{
+// Alignment != natural
+E(InvalidAlignment, 0x0200, "atomic alignment must be equal to natural")
 // Alignment > natural
-E(InvalidAlignment, 0x0200, "alignment must not be larger than natural")
+E(AlignmentTooLarge, 0x0201, "alignment must not be larger than natural")
```

Comment:
@q82419 Should we shift these many error code?

Reply:
I think its Okay, because we always suggest developers to use enum rather than code.